stores/gdrive/authorization-model.json

```diff
@@ -148,6 +185,34 @@
             ]
           }
         }
+      },
+      "metadata": {
+        "relations": {
+          "can_change_owner": { "directly_related_user_types": [] },
+          "owner": {
+            "directly_related_user_types": [{
+                "type": "user"
+            }]
+          },
+          "parent": {
+            "directly_related_user_types": [{
+                "type": "folder"
+            }]
+          },
+          "can_read": { "directly_related_user_types": [] },
+          "can_share": { "directly_related_user_types": [] },
+          "viewer": {
+            "directly_related_user_types": [{
+                "type": "user"
+            }, {
+                "type": "user:*"
```

Comment:
I supposed we don't need user:* now, right?

Reply:
Fixed